examples/research_projects/jax-projects/HOW_TO_PROPOSE_PROJECT.md

```diff
@@ -0,0 +1,97 @@
+# How to propose a Flax/JAX + Transformers project 
+
+Great that you've opened this document! 
+While we at 🤗 are proposing a couple of projects, we strongly 
+believe that the community can come up with much more **creative**, **fun**, and 
+**impactful** projects on their own. This being said, we are really looking forward
+to see your project proposal! 
+
+## What a project should be about
+
+The proposed project should fall into the machine learning fields of **Natural Language Processing (NLP)** and/or **Computer Vision (CV)** (possibly also **Speech Recognition (ASR)** depending on whether Speech Recognition models are available in Flax in due time) and aim at solving a specific task. 
+Possible tasks can belong to text classification, text generation, image recognition, image processing, image captioning, audio classification, etc. 
+The clearer a task is defined, the better your project proposal is.
+*E.g.* "Using a T5 model to learn grammar correction in French" or "Adapting a pre-trained CLIP model for zero-shot image classification in Spanish" are **well-defined and clear** project proposals, while something like "Train a language model" or "Image classification" are **too vague**.
+There is no limit to your creativity as long as the project is feasible and ethical.
+The more creative & specific your project proposal, the more interesting it will be, 
+and the more likely will you find motivated team members to work on your project!
+To get an idea of how to formulate your project proposals, you can browse through 
+existing project proposals on the [forum](https://discuss.huggingface.co/c/flax-jax-projects/22).
```

Comment:
I would suggest to try to avoid these large paragraphs or people won't read all of it